samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/AssessmentGradingFacadeQueries.java

```diff
@@ -2253,6 +2253,8 @@ public List getExportResponsesData(String publishedAssessmentId, boolean anonymo
 				  boolean addRationale = false;
 				  boolean addResponseComment = false;
 
+				  boolean matrixChoices = false;
+				  TreeMap responsesMap = new TreeMap();
```

Comment:
I dont think a TreeMap is warranted here, unnecessary overhead. LinkedHashmap would suffice if order is required.